ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -2316,7 +2323,55 @@
         {
             "cell_type": "code",
             "source": [
-                "public interface ComputeIfrsVarsOpenings : ActualToIfrsVariable, DeferrableToIfrsVariable, TmToIfrsVariable",
+                "public interface ProjectionScopeCashflows : IScope<ImportIdentity, ImportStorage>{",
+                "\n    int upperRange => ImportStorage.DefaultProjectionCount;",
+                "\n    IEnumerable<IfrsVariable> CalculatedIfrsVariables => Enumerable.Range(0,upperRange).Select(x => GetScope<ComputeIfrsVarsCashflows>(Identity with { ProjectionPeriod = x} ).CalculatedIfrsVariables //Here Projection Period needs to be x...and then the right information needs to be found. ",
+                "\n                                                            .Select(y => AdjustProjection(y, x))).SelectMany(l => l).AggregateOver().Select(x => x with {Partition = GetStorage().TargetPartition});",
+                "\n",
+                "\nIfrsVariable AdjustProjection(IfrsVariable iv, int projectionPeriod){",
```

Comment:
Should I bring this method to that NB, to the "Import helpers" chapter?

Reply:
@dcolleoni No, Davide, it wouldn't be possible to move it to the "Import helpers" as discussed above because the Importers.nb is importing ImportScopeCalculation.nb at the first place, not the other way around. So I suggest to bring it to the ifrs17/Utils/ImportCalculationMethods.nb. Do you agree, Davide?

Reply:
Done.